requirements/common_constraints.txt

```diff
@@ -13,8 +13,13 @@
 
 
 # using LTS django version
-Django<4.0
+Django<3.3
```

Comment:
shouldn't this constraint be `Django<4.0`

Reply:
make upgrade fetched it from common constraints. I have a pending PR that needs to be merged to resolve this. I will get that merged now.